deployments/kubernetes/chart/reloader/templates/deployment.yaml

```diff
@@ -73,9 +73,13 @@ spec:
       priorityClassName: {{ .Values.reloader.deployment.priorityClassName }}
       {{- end }}
       containers:
-      - image: "{{ .Values.reloader.deployment.image.name }}:{{ .Values.reloader.deployment.image.tag }}"
+      - name: {{ template "reloader-fullname" . }}
+        {{- .Values.reloader.deployment.image.digest }}
```

Comment:
doesnt this condition need an if statement?

Reply:
good catch! fixed